src/main/java/com/securian/creditcompass/entities/Claim.java

```diff
@@ -162,7 +163,11 @@ private void notifyStateChangeListeners(ClaimState oldState, ClaimState newState
     public void setClaimExaminer(ClaimsExaminer examiner) {
         this.examiner = examiner.getUsername();
     }
+
+    //This setter should only be used for testing purposes
+    public void setClaimStateTesting() {this.currentState = new AssignedClaimState();
```

Comment:
No longer needed since the 'States' are redundant (as described by Pratibha in her review)

Reply:
Thanks for the feedback